libs/interfaces/src/lib/design/design.interface.ts

```diff
@@ -50,3 +51,9 @@ export interface IFamilyTreeBanner {
   text: string;
   style: 'first';
 }
+
+export interface Template {
```

Comment:
interfaces should reflect their existance in the variable name
```suggestion
export interface ITemplate {
```
This change has to be applied to all instanced of Template

You should also change the name to something more descriptive like `IFamilyTreeTemplate`